diff --git a/toolkit/moz.configure b/toolkit/moz.configure
index bfb5a38258..1c683cb6ec 100644
--- a/toolkit/moz.configure
+++ b/toolkit/moz.configure
@@ -22,7 +22,7 @@ def check_moz_app_id(moz_app_id, build_project):
 
 project_flag(
     env="MOZ_APP_VENDOR",
-    default="Zen Team",
+    default="NevoFlux Team",
     nargs=1,
     help='Used for application.ini\'s "Vendor" field, which also impacts profile location and user-visible fields',
 )
@@ -36,7 +36,7 @@ project_flag(
 
 project_flag(
     "MOZ_APP_PROFILE",
-    default="zen",
+    default="nevoflux",
     nargs=1,
     help='Used for application.ini\'s "Profile" field, which controls profile location',
 )
